scripts/apps/archive/controllers/UploadController.ts

```diff
@@ -328,18 +335,32 @@ export function UploadController(
                 notify.error(message);
             }
 
-            if (invalidFiles.length > 0) {
-                notify.error(
-                    gettext(
-                        `The image you\'re trying to add is smaller than
-                        {{width}} x {{height}} pixels. Please use another one. File name: {{filename}}`,
+            if (invalidImages.length > 0) {
+                const images = invalidImages.map((image) => {
+                    return gettext(
+                        '<li>The size of {{name}} is {{width}}x{{height}}</li>',
                         {
-                            width: appConfig.pictures.minWidth,
-                            height: appConfig.pictures.minHeight,
-                            filename: invalidFiles.join(', '),
+                            name: image.name,
+                            width: image.width,
+                            height: image.height,
+                        },
+                    );
+                });
+
+                modal.alert({
+                    headerText: gettextPlural(
+                        images.length,
+                        `The image you\'re trying to add is smaller than {{minWidth}}x{{minHeight}}
```

Comment:
It's better to keep titles short and explain the issue in detail in the modal body. Something like "(Some) files were not uploaded" or "Uploading error" would be enough for the title.

It would also be useful to mention that other files were successfully uploaded and only the ones with errors have to be reuploaded, given that's how it works.